packages/react/src/components/Storage/FileUploader/UploadPreviewer/index.tsx

```diff
@@ -18,10 +18,13 @@ export function UploadPreviewer({
   isLoading,
   isSuccessful,
   hasMaxFilesError,
+  maxFiles,
   onClear,
   onFileClick,
 }: PreviewerProps): JSX.Element {
-  const headingMaxFiles = translate('Over Max files');
+  const headingMaxFiles = `${translate('Over')} ${maxFiles} ${translate(
+    ' file limit'
+  )}`;
```

Comment:
When I redo the translations this will be updated.

Reply:
+1 [11394eb](https://github.com/aws-amplify/amplify-ui/pull/3344/commits/11394ebebbc792ddb8299e074c7569ad2e35a103)
Merge state
Add more commits by pushing to the [fileuploader/over-max-file-error](https://github.com/aws-amplify/amplify-ui/tree/fileuploader/over-max-file-error) branch on [aws-amplify/amplify-ui](https://github.com/aws-amplify/amplify-ui).

This branch was previously deployed
1 inactive (outdated) deployment
Code owner review required
Waiting on code owner review from aws-amplify/amplify-ui. 
1 pending reviewer
Some checks haven’t completed yet
2 in progress, 2 successful, and 11 expected checks
@github-actions
CodeQL / Analyze (javascript) (pull_request) In progress — This check has started...
[Details](https://github.com/aws-amplify/amplify-ui/actions/runs/4001669743/jobs/6868141330)
@github-actions
Test / PRs / setup (pull_request_target) Successful in 2s
[Details](https://github.com/aws-amplify/amplify-ui/actions/runs/4001669618/jobs/6868141049)
@github-actions
Test / PRs / dependency-review (pull_request_target) Successful in 7s
[Details](https://github.com/aws-amplify/amplify-ui/actions/runs/4001669618/jobs/6868141225)
@github-actions
Test / PRs / test / setup (pull_request_target) In progress — This check has started...
[Details](https://github.com/aws-amplify/amplify-ui/actions/runs/4001669618/jobs/6868142991)
CodeQL Expected — Waiting for status to be reported
Required
test / docs Expected — Waiting for status to be reported
Required
test / e2e (angular, angular, @angular and not @todo-angular) Expected — Waiting for status to be reported
Required
test / e2e (next, react, @react and not @todo-react) Expected — Waiting for status to be reported
Required
test / e2e (vue, vue, @vue and not @todo-vue) Expected — Waiting for status to be reported
Required
test / unit (angular) Expected — Waiting for status to be reported
Required
test / unit (react) Expected — Waiting for status to be reported
Required
test / unit (react-core) Expected — Waiting for status to be reported
Required
test / unit (react-native) Expected — Waiting for status to be reported
Required
test / unit (ui) Expected — Waiting for status to be reported
Required
test / unit (vue) Expected — Waiting for status to be reported
Required
This branch is out-of-date with the base branch
Merge the latest changes from main into this branch.
This merge commit will be associated with icystorm@gmail.com.

Merging is blocked
Merging can be performed automatically with 2 approving reviews.
 Automatically merge when all requirements are met. [Learn more](https://docs.github.com/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request)
@ErikCH

 
Add heading textAdd bold text, <Cmd+b>Add italic text, <Cmd+i>
Add a quote, <Cmd+Shift+.>Add code, <Cmd+e>Add a link, <Cmd+k>
Add a bulleted list, <Cmd+Shift+8>Add a numbered list, <Cmd+Shift+7>Add a task list, <Cmd+Shift+l>
Directly mention a user or team
Reference an issue, pull request, or discussion
Add saved reply
Leave a comment
No file chosen
Attach files by dragging & dropping, selecting or pasting them.
Styling with Markdown is supported
Remember, contributions to this repository should follow its [contributing guidelines](https://github.com/aws-amplify/amplify-ui/blob/68c0a0e1c89ede36b4b0bdd7aad8f82f8c5b23b7/CONTRIBUTING.md), [security policy](https://github.com/aws-amplify/.github/blob/677e2710f7c81179e1b20e2f014a1b96ef0cbc35/SECURITY.md), and [code of conduct](https://github.com/aws-amplify/amplify-ui/blob/68c0a0e1c89ede36b4b0bdd7aad8f82f8c5b23b7/.github/CODE_OF_CONDUCT.md).
 ProTip! Add comments to specific lines under [Files changed](https://github.com/aws-amplify/amplify-ui/pull/3344/files).
Reviewers
@[wlee221](https://github.com/wlee221)
wlee221
@aws-amplify/amplify-ui
amplify-ui
At least 2 approving reviews are required to merge this pull request.

Still in progress? 
Assignees
No one—
Labels
None yet
Projects
None yet
Milestone
No milestone
Development
Successfully merging this pull request may close these issues.

None yet

Notifications
Customize
You’re receiving notifications because you authored the thread.
2 participants
@ErikCH
@wlee221